src/main/java/eu/dissco/dataexporter/component/JobSchedulerComponent.java

```diff
@@ -0,0 +1,84 @@
+package eu.dissco.dataexporter.component;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.dissco.dataexporter.domain.ExportJob;
+import eu.dissco.dataexporter.exception.SchedulingFailedRuntimeException;
+import eu.dissco.dataexporter.properties.JobProperties;
+import eu.dissco.dataexporter.repository.DataExporterRepository;
+import eu.dissco.dataexporter.schema.SearchParam;
+import freemarker.template.Configuration;
+import freemarker.template.Template;
+import freemarker.template.TemplateException;
+import io.kubernetes.client.openapi.apis.BatchV1Api;
+import io.kubernetes.client.openapi.models.V1Job;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor
+public class JobSchedulerComponent {
+
+  private final DataExporterRepository repository;
+  private final JobProperties jobProperties;
+  private final Template jobTemplate;
+  private final BatchV1Api batchV1Api;
+  @Qualifier("yamlMapper")
+  private final ObjectMapper yamlMapper;
+
+  @Scheduled(fixedRate = 3600)
+  public void schedule(){
+    var runningJobs = repository.getRunningJobs();
+    if (runningJobs < jobProperties.getQueueSize()){
+      var nextJob = repository.getNextJobInQueue();
+      nextJob.ifPresent(this::scheduleJob);
+    }
+  }
+
+  private void scheduleJob(ExportJob exportJob) {
+    var job = createV1Job(exportJob);
+    batchV1Api.createNamespacedJob(jobProperties.getNamespace(), job);
```

Comment:
Maybe add some happy logging here to indicate we successfully launched a new job